src/components/todoItems.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function ToDoItems({ toDos, onClick }) {
+  function getToDoListView() {
+    if (toDos.length <= 0) {
+      return <p>할 일이 없어요!</p>;
+    }
+    return (
+      toDos.map((todo) => (
+        <li>
+          {todo}
+          <button type="button" key={todo} onClick={() => { onClick(todo); }}>
+            완료
+          </button>
+        </li>
+      ))
+    );
+  }
+
+  return (
+    <ol type="1">
+      {getToDoListView()}
+    </ol>
+  );
```

Comment:
```suggestion
  if (toDos.length <= 0) {
    return <p>할 일이 없어요!</p>;
  }

  return (
    <ol type="1">
      {toDos.map((todo) => (
        <li>
          {todo}
          <button type="button" key={todo} onClick={() => { onClick(todo); }}>
            완료
          </button>
        </li>
      ))
      }
    </ol>
  );
```

할 일이 없는 경우에 대해서 잘 분리해주셨네요👍🏼  다만 함수로 한번 더 감싸서 처리하는 것 보다 위와 같이 처리하는건 어떨까요?